tasks/dns-records.yml

```diff
@@ -1,10 +1,21 @@
 ---
-- debug: var=i_zone
+- name: debug changes
+  debug: var=i_zone
 
-- name: sleep for 10 seconds and continue with play
-  wait_for: timeout=10
+- name: Confirm changes
+  pause:
+    prompt: 'Please review the changes in the last task and press enter to continue'
+  when: check_route53_zones.send_to_aws | length != 0 and force | default("false") != "true"
   no_log: true
 
+- name: notify
+  include: notify.yml
+  vars:
+    notify_message: "{{ check_route53_zones.json_diff | to_nice_json }}"
```

Comment:
What about send `{{ ansible_user }}` to notification chat?

Reply:
Done 👍